db/src/fakes/github.js

```diff
@@ -18,6 +19,7 @@ class FakeGithub {
     this.taskclusterIntegrationOwners = new Map();
     this.taskclusterChecksToTasks = new Map();
     this.taskclusterCheckRuns = new Map();
+    this.github_builds = new Map();
```

Comment:
Let's use camel case to align with JS conventions and the other names in this file.